meson.build

```diff
@@ -39,13 +41,7 @@ pixman = dependency('pixman-1')
 libturbojpeg = dependency('libturbojpeg', required: get_option('tight-encoding'))
 gnutls = dependency('gnutls', required: get_option('tls'))
 zlib = dependency('zlib')
-
```

Comment:
Because I have packed `aml` as another package and if I also have aml here it will not use the system installed version.

It is better to have different software built as different packages to avoid version discrepancies and also to know exactly what version of each package is installed.

I am fine with putting it back but in that case I would like to have an option available, something like `system-aml` or `local-aml`. Which will enforce which one of them gets use.

Reply:
done.

Reply:
Isn't it enough to just not have the subproject in the directory?